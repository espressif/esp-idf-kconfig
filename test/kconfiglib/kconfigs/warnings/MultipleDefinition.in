mainmenu "Symbol defined multiple times"

    config FOO
        int "Foo"
        default 1 if NOT_DEFINED
        default 0

    config FOO
        int "Second Foo"
        default -1
