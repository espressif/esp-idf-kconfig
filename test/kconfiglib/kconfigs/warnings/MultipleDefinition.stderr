Status: Finished with notifications
Multiple definitions
FOO
